src/aliceVision/sensorDB/cameraSensors.db

```diff
@@ -2008,9 +2008,10 @@ Gome;Gome K1;4.74;devicespecifications
 Gome;Gome U7;4.73;devicespecifications
 Google;Google Pixel;6.25;devicespecifications
 Google;Google Pixel 2;5.5;imaging-resource
-Google;Google Pixel 2 XL;5.5;devicespecifications
+Google;Pixel 2 XL;5.5;devicespecifications
```

Comment:
Are you sure it really needs `Google` to be removed?

Normally, the comparison algorithm is robust enough to match that, i.e. see 
https://github.com/alicevision/AliceVision/blob/develop/src/aliceVision/sensorDB/Datasheet.cpp#L49
if you have, e.g.  `modelA = Pixel 2 XL` and `modelB = Google Pixel 2 XL` they won't be the same (`modelA == modelB`) but one of the `boost::algorithm::ends_with(modelA, modelB)` should still be true.

Please try with the `Brand Model` version, for coherence it would be better to use that as all the others are like that.

Reply:
Hi,
i think it still takes the correct values but it gives this warning that suggests to correct the entry in the database:
```
[12:46:58.450651][warning] Some image(s) have no serial number to identify the camera/lens device.
This makes it impossible to correctly group the images by device if you have used multiple identical (same model) camera devices.
The reconstruction will assume that only one device has been used, so if 2 images share the same focal length approximation they will share the same internal camera parameters.
50 image(s) are concerned.
[12:46:58.451063][warning] The camera found in the database is slightly different for image(s):
[12:46:58.451075][warning] image: 'photo5346092996659407635.jpg'
	- image camera brand: Google
	- image camera model: Pixel 3 XL
	- database camera brand: Google
	- database camera model: Google Pixel 3 XL
	- database camera sensor size: 5.76 mm
[12:46:58.451092][warning] Please check and correct camera model(s) name in the database.
```

Reply:
Yes, it's just a warning to be sure that it is not using a completely different model.
Ok, let's keep it this way then.
Thanks for your contribution!